pkg/cloudprovider/huaweicloud/huaweicloud.go

```diff
@@ -456,6 +356,11 @@ func NewHWSCloud(cfg io.Reader) (*CloudProvider, error) {
 		return nil, err
 	}
 
+	err = hws.securityGroupListener()
```

Comment:
After starting monitoring, when will it stop?
Please refer to the original start and stop methods